source/_components/toon.markdown

```diff
@@ -77,3 +77,7 @@ The `toon` climate platform allows you to interact with your Toon thermostat. Fo
 | Heat           | Thuis   |
 | Eco            | Weg     |
 | Cool           | Slapen  |
+
+It also supports setting the temperature manually.
+
+The Toon API is polled at a 30-second interval, so the status is relatively fresh without overloading the API.
```

Comment:
```suggestion
<p class='note'>
The Toon API is polled at a 30-second interval, so the status is relatively fresh without overloading the API.
</p>
```

Reply:
Not sure if we need to place the 30 seconds interval in a note here.

You can change temperatures by calling operations modes. Setting a specific temperature, however, fails. This is caused by a larger issue on the HA platform, which is currently under discussion as an architectural issue. Adding this note here, would require adding it in a lot more places? Not sure about that.